docs/mathematics/Matrix-Addition.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: Matrix Addition
+slug: /mathematics/Matrix-Addition
+---
+
+## Introduction
+
+In mathematics, matrix addition is the operation of adding two matrices by adding the corresponding entries together. Two matrices must have an equal number of rows and columns to be added.
+
+## Explanation
+
+Consider two matrices A and B , the sum of two matrices A and B will be a matrix which has the same number of rows and columns as A and B. The sum of A and B, denoted A + B, It is computed by adding corresponding elements of A and B
+
+
+## Algorithm
+
+Input matrix A and matrix B.<br/>
+Input matrix 1 <br/>
+Input matrix 2 <br/>
+If the number of rows and number of columns of matrix A and matrix B is equal<br/>
+for i=0 to number of rows in A do<br/>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;for j=0 to number of rows in A do<br/> 
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;        Result [i,j] =  A [i,j]+ B [i,j]<br/>
+Display Result
```

Comment:
Keep in a bullet list. Use the algorithm just for stating the steps. Don'r use any code/pseudo-code here